lib/src/main/scala/code/winitzki/jc/JoinRun.scala

```diff
@@ -14,7 +14,6 @@ import java.util.concurrent.{Semaphore, TimeUnit}
 
 import scala.collection.mutable
 import scala.concurrent.duration.Duration
-import scala.reflect.ClassTag
```

Comment:
I want to avoid using reflection as much as possible. Tpolecat (Scalaz) says code should be invariant to class name renaming and reflection typically breaks this. Travis Brown quoted him in his Circe JSON parser project.